torch/ao/nn/quantized/modules/conv.py

```diff
@@ -833,7 +833,7 @@ def from_float(cls, mod, use_precomputed_fake_quant=False):
         ):
             return qconv  # dynamic quantization doesn't need scale/zero_point
         else:
-            act_scale, act_zp = mod.activation_post_process.calculate_qparams()
+            act_scale, act_zp = mod.activation_post_process.calculate_qparams()  # type: ignore[operator, union-attr]
```

Comment:
We may want add a typing utility like we have for not_none to assert things are callable.